content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md

```diff
@@ -415,3 +415,14 @@ If `/var/lib/kubelet` is being mounted, performing a `kubeadm reset` will effect
 To workaround the issue, re-mount the `/var/lib/kubelet` directory after performing the `kubeadm reset` operation.
 
 This is a regression introduced in kubeadm 1.15. The issue is fixed in 1.20.
+
+## Cannot use the metrics-server securely in a kubeadm cluster
```

Comment:
I'm not sure about this title. Clearly you _can_ use the metrics-server in a kubeadm cluster, including encryption in transit.

What's the warning that someone sees if they haven't read this advice? Could we use that warning as the basis for the title?

Reply:
as per the kubeadm default setup, one can only use the metrics-server if they pass `--kubelet-insecure-tls`.

due to the default self-signed serving certificate for the kubelet they will get errors like:

> x509: certificate signed by unknown authority
> x509: certificate is valid for IP-foo not IP-bar

we can include these two as symptoms.

even if the certificate is properly signed, i don't know what errors happen later if TLS related flags are not properly configured for the metrics server.

the title seems fine to me, so if you think something is better could you recommend an exact replacement?